code/src/components/MessageInput.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from 'react'
+import './messageInput.css'
+
+export const MessageInput = () => {
+  const MESSAGES_URL = "https://technigo-thoughts.herokuapp.com";
+  const [message, setMessage] = useState("");
+
+  const handleClick = event => {
+    event.preventDefault();
+
+    fetch(MESSAGES_URL, 
+      {
+        method: "POST",
+        headers: {
+          'Content-Type':'application/json'
+        },
+        body:JSON.stringify({message})
+      }
+    ).then(()=>{
+      window.location.reload();
+    })
+  };
+  
+  return (
+    <section className="input-section">
+      <form className="message-card">
+        <p>What's making you happy right now?</p>
+        <textarea className="textarea"
+          rows= '2'
+          type="text"
+          onChange={event => setMessage(event.target.value)}>
+        </textarea>
+        <h4>{140 - message.length}</h4>
```

Comment:
Hm I really don't think this needs to be a title. I instead would like to have the line 27 as a title and this as a paragraph. What do you think about that?

Reply:
Very good point! Will swap!